src/routes/users.js

```diff
@@ -0,0 +1,8 @@
+const express = require('express');
+
+const { handleValidation } = require('../utility/validation');
+
+const router = express.Router();
+const UsersController = require('../controllers/users');
+
+router.post('/', handleValidation, UsersController.addUserSubscription);
```

Comment:
Where is the :id and the the sub in this endpoint? 
Shouldn't this be like `/:id/sub`?